projects/control-service/projects/pipelines_control_service/src/test/java/com/vmware/taurus/service/KubernetesServiceTest.java

```diff
@@ -309,12 +416,19 @@ public void testQuantityToMbConversion(int expectedMb, String providedResources)
 
     private KubernetesService mockCronJobFromTemplate() {
        KubernetesService mock = Mockito.mock(KubernetesService.class);
-       Mockito.when(mock.cronJobFromTemplate(
+       Mockito.when(mock.getK8sSupportsV1CronJob()).thenReturn(false);
```

Comment:
it seems the getter is mocked, however implementation seems to use the field directly,
for example `this.k8sSupportsV1CronJob` usages in KubernetesService.java?
This reference is also redundant btw

Reply:
Thanks for the note, I fixed the references to this.k8sSupportsV1CronJob in KubernetesService and replaced them with the getter.